test/e2e/antctl_test.go

```diff
@@ -0,0 +1,144 @@
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// antctlOutput is a helper function for logging antctl outputs.
+func antctlOutput(stdout, stderr string, tb testing.TB) {
+	tb.Logf("antctl stdout:\n%s", stdout)
+	tb.Logf("antctl stderr:\n%s", stderr)
+}
+
+// runAntctl runs antctl commands on antrea pods, the controller, or agents. It
+// always runs the commands with verbose flag enabled.
+func runAntctl(podName string, subCMDs []string, data *TestData, tb testing.TB) (string, string, error) {
+	var containerName string
+	if strings.Contains(podName, "agent") {
+		containerName = "antrea-agent"
+	} else {
+		containerName = "antrea-controller"
+	}
+	cmds := []string{"antctl", "-v"}
+	stdout, stderr, err := data.runCommandFromPod(antreaNamespace, podName, containerName, append(cmds, subCMDs...))
+	antctlOutput(stdout, stderr, tb)
+	return stdout, stderr, err
+}
+
+// TestAntctlAgentLocalAccess ensures antctl is accessible in agent pod.
```

Comment:
s/pod/Pod (here and other places in the file)

Reply:
Done.